contrib/cirrus/lib.sh

```diff
@@ -224,16 +224,19 @@ install_testing_deps() {
         GOPATH $GOPATH
         GOSRC $GOSRC
     "
+
     echo "Installing required go packages into \$GOPATH"
     for toolpath in \
         tools/godep \
         onsi/ginkgo \
         onsi/gomega \
         cloudflare/cfssl/cmd/... \
         jteeuwen/go-bindata/go-bindata \
-        cpuguy83/go-md2man
+        cpuguy83/go-md2man \
+        urfave/cli \
+        containers/image/storage
     do
-        go get -u "github.com/$toolpath"
+        go get -d "github.com/$toolpath"
```

Comment:
I'm no go-expert, but from the `--help` it says `-u` will update and install missing packages, where as `-d` does NOT do the install.  Do the packages need "installing"?